jsk_topic_tools/CMakeLists.txt

```diff
@@ -98,6 +98,8 @@ if (NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
   add_rostest(test/test_hz_measure.test)
   add_rostest(test/test_block.test)
 endif(NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
```

Comment:
So you mean the catkin version on `hydro` is newer than that on `indigo`?
I wonder why `NOT ... STREQUAL`.